test/utils/helpers.ts

```diff
@@ -119,3 +121,18 @@ export function isAddressInfo(address: unknown): address is net.AddressInfo {
     (address as net.AddressInfo).address !== undefined
   )
 }
+
+/**
+ * If using a proxy, return the address of the proxy.
+ *
+ * Otherwise, return the direct address of code-server.
+ */
+export async function getMaybeProxiedCodeServer(codeServer: CodeServerPage | CodeServer): Promise<string> {
+  let address = await codeServer.address()
+  if (process.env.USE_PROXY && process.env.USE_PROXY === "1") {
```

Comment:
nit: we could do with just the latter check

```suggestion
if (process.env.USE_PROXY === "1") {
```

Reply:
Ooops sorry my suggestion removed the indent, I keep forgetting copy does not copy the indent and I have to add it back in manually.

Reply:
😂 i'll fix locally, no worries. i didn't realize that eitehr